docs/guides/universal-profile/01-create-profile.md

```diff
@@ -226,20 +228,21 @@ const lspFactory = new LSPFactory(
     deployKey: PRIVATE_KEY, // or myEOA.privateKey
   },
 );
-// LSPFactory {
-//   options: {
-//     ...
-//   },
-//   DigitalAsset: DigitalAsset {
-//       ...
-//   },
-//   LSP3UniversalProfile: LSP3UniversalProfile {
-//       ...
-//   },
-//   ProxyDeployer: ProxyDeployer {
-//       ...
-//   }
-// }
+
+> LSPFactory {
+  options: {
+    ...
+  },
+  DigitalAsset: DigitalAsset {
+      ...
+  },
+  LSP3UniversalProfile: LSP3UniversalProfile {
+      ...
+  },
+  ProxyDeployer: ProxyDeployer {
+      ...
+  }
+}
```

Comment:
im not sure if this is needed?

Or if so, add:

console.log(lspFactory)

/**
... the output ...
*/

?